### PR TITLE
add doc for CSIServiceAccountToken

### PR DESCRIPTION
**Pull request by zshihang**
enhancement issue: https://github.com/kubernetes/enhancements/issues/2047
KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1855-csi-driver-service-account-token

### CODE REVIEW

**Comment by annajung** [comment]
/hold 
pending merge of https://github.com/kubernetes/kubernetes/pull/93130

/milestone 1.20
/assign @eagleusb 
/sig auth storage

**Comment by sftim** [comment]
Hi @zshihang 

The earlier you can provide some content for documenting this feature, the earlier that SIG Docs can get reviewing. We'd really appreciate a chance to get as much reviewing done as we can before KubeCon.

**Comment by zshihang** [comment]
the doc will be added to this repo https://kubernetes-csi.github.io/docs. the doc change for kubernetes.io is just the feature gate. cc @msau42

**Comment by sftim** [comment]
It might be useful to add a task page about configuring service account tokens (and without making security slips). What do you think @zshihang ?

If that's true, it might well be OK to track that need as an issue rather than delay shipping the feature. But, let's still track it.

**Comment by zshihang** [comment]
that's a good idea. this service account token is particularly for csi driver so the csi website would be a good place to start.

**Review by irvifa — COMMENTED** [review]
/unhold
K/k is merged

**Comment by zshihang** [comment]
added the doc in csi website https://github.com/kubernetes-csi/docs/pull/383

**Comment by annajung** [comment]
Thanks @zshihang 

/lgtm
/approve

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ different Kubernetes components.
 | `CSIMigrationOpenStackComplete` | `false` | Alpha | 1.17 | |
 | `CSIMigrationvSphere` | `false` | Beta | 1.19 | |
 | `CSIMigrationvSphereComplete` | `false` | Beta | 1.19 | |
+| `CSIServiceAccountToken` | `false` | Alpha | 1.20 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:82]: Looks to me this new gate is not documented?
We need a short description for each feature gate on this page. Please see the last section.

Comment (annajung) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:82]: @zshihang looks like I missed this before merging, could you open up another PR for description?

Comment (zshihang) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:82]: https://github.com/kubernetes/website/pull/25028

**Pull request merged**